app/src/ai/components/ai-sidebar-detail.vue

```diff
@@ -0,0 +1,136 @@
+<script setup lang="ts">
+import VIcon from '@/components/v-icon/v-icon.vue';
+import { useShortcut } from '@/composables/use-shortcut';
+import { translateShortcut } from '@/utils/translate-shortcut';
+import { useSidebarStore } from '@/views/private/private-view/stores/sidebar';
+import { useElementHover } from '@vueuse/core';
+import { CollapsibleContent, CollapsibleRoot, CollapsibleTrigger } from 'reka-ui';
+import { useTemplateRef } from 'vue';
+import { useAiStore } from '../stores/use-ai';
+import AiConversation from './ai-conversation.vue';
+import AiMagicButton from './ai-magic-button.vue';
+
+const sidebarStore = useSidebarStore();
+const aiStore = useAiStore();
+
+const aiSidebarCollapsibleTriggerContent = useTemplateRef('collapsible-trigger-content');
+
+const hovering = useElementHover(aiSidebarCollapsibleTriggerContent);
+
+useShortcut('meta+j', () => {
+	aiStore.chatOpen = !aiStore.chatOpen;
+});
+</script>
+
+<template>
+	<CollapsibleRoot
+		v-model:open="aiStore.chatOpen"
+		v-tooltip.left="!aiStore.chatOpen && `${$t('ai_chat')} (${translateShortcut(['meta', 'j'])})`"
+		class="collapsible-root"
+	>
+		<CollapsibleTrigger class="collapsible-trigger">
+			<div ref="collapsible-trigger-content" class="collapsible-trigger-content">
+				<AiMagicButton class="collapsible-trigger-icon" :animate="hovering" />
+				<span v-show="!sidebarStore.collapsed" class="collapsible-trigger-title">{{ $t('ai_chat') }}</span>
+				<v-chip v-show="!sidebarStore.collapsed" outlined primary x-small class="collapsible-trigger-beta">
+					{{ $t('beta') }}
+				</v-chip>
+				<VIcon v-show="!sidebarStore.collapsed" name="chevron_left" class="collapsible-trigger-chevron" />
+			</div>
+		</CollapsibleTrigger>
+		<CollapsibleContent class="collapsible-content">
+			<div class="ai-sidebar-content">
+				<ai-conversation />
+			</div>
+		</CollapsibleContent>
+	</CollapsibleRoot>
+</template>
+
+<style scoped>
+.collapsible-root {
+	inline-size: 100%;
+	background-color: var(--theme--sidebar--background);
+}
+
+.collapsible-trigger {
+	block-size: 60px;
+	inline-size: 100%;
+}
```

Comment:
Let’s adjust the focus ring so that it matches the other sidebar buttons.

```suggestion
.collapsible-trigger {
    --focus-ring-offset: var(--focus-ring-offset-invert);

	block-size: 60px;
	inline-size: 100%;
}
```